app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,10 @@
         <activity
             android:name=".ProfilePageActivity"
             android:parentActivityName=".activities.ProjectsListActivity"/>
+        <activity
+            android:name=".video.VideoSubtitlingActivity"
+            android:screenOrientation="portrait" />
+        <activity android:name=".ProfilePageActivity" />
```

Comment:
The build failed because there is two ProfilePageActivity in the AndroidManifest.
```suggestion
```